### ISSUE TITLE
Bundling with new RetroArch release?

### ISSUE THREAD

**Issue by inactive123** [meancoot/MFiWrapper#10]
Hi there,

I'm planning a new release of RetroArch for next week or so. How far along is the MFIWrapper and can it be used to successfully sync and play games with now for both iOS 6/7?

If further testing is required I can help out.

**Comment by OMGsus** [comment]
I have not tested ios6 yet, but , other than that... On the mfiwrapper side, wii based controls work great. Ds4 works for some people but not me. Mixing with real mfi controllers causes a crash in gccontroller apps. (Open issues). 

As far as how it works once a controller is synced. So far so good. RA picks it up nicely. Multiplayer works well to and player assignment works as expected. You would think the controller was a real mfi controller. Being on the native stack is also a HUGE plus. 

On the RA side, support for the full extension controls (start select r3 / l3) needs to be handled. I started to add it but I'm slow and don't want to step on any toes. 

Also, the gcpause thingy state seems broken (bound to home/ps buttons) I'm not sure if this is a RA issue or mfiwrapper issue as both seem to have the code to handle it. 

All Just my 0.2¢ as a tester 

As far as testing. The more the merrier. Not sure if anyone else has tested with ios6.

**Comment by inactive123** [comment]
@meancoot I have released RA now for all other systems except for iOS. How far along is MFiWrapper? Can I already bundle it with RetroArch or would this be premature?

Looking forward to hearing from you.

**Comment by nintyapple** [comment]
@twinaphex @meancoot Any plans on this? I know you released iOS version without MFiWrapper, so I wondered when they might be bundled?

**Comment by JoshSchreuder** [comment]
@twinaphex @meancoot will this be bundled with RA in the near future?

**Comment by nintyapple** [comment]
@JoshSchreuder No: RetroArch made a blog post saying that meancoot has gone missing, so they are instead making BTStack patches to RetroArch, so it works properly with BTStack. Along with this, the developer of the tweak, Controllers For All, added MFi support to RetroArch, so when RetroArch 1.0.0.3. releases it will have MFi and BTStack working properly so you'll be able to use Controllers For All instead of MFiWrapper (you'll also have to wait for Controllers For All 1.4 with official RetroArch support.
